app/views/effective/classifieds/index.html.haml

```diff
@@ -1,9 +1,38 @@
+- classifieds = ::Effective::Classified.classifieds(user: current_user).paginate(page: params[:page], per_page: 12)
+
 = render 'layout' do
   .resource-buttons
     = link_to "Submit New #{et(Effective::Classified)}", effective_classifieds.new_classified_wizard_path, class: 'btn btn-primary'
 
-  = card do
-    - if @datatable.present?(self)
-      = render_datatable(@datatable, simple: true)
-    - else
-      %p There are no active #{etsd(Effective::Classified)}. When there are, we'll show them here.
+  - if classifieds.present?
+    .actions.d-flex.flex-row-reverse.my-3
```

Comment:
This is only for that Grid/List view option, we can remove but think about adding it and seeing what are your feelings about it?